.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/passfort
+    docker:
+      - image: ruby:2.5
```

Comment:
What we do in our other libraries is to have a matrix of builds across different ruby versions, but I've not bothered with this for now in the interests of time. When we upgrade to 2.6 we should keep tests running against 2.5 in CI so that we don't introduce any regressions.